### ISSUE TITLE
Film stop at 2 minutes

### ISSUE THREAD

**Issue by titouancreach** [titouancreach/peerflix-gui#1]
The movies stop about 2 minutes,
Nodejs probably stop the movie after no responses from the child process.

**Comment by titouancreach** [comment]
```
child = exec("firefox", (error, stdout, stderr) => {
  console.log(stdout);
});
```

works over 10 minutes

```
child = exec("cat /dev/urandom", (error, stdout, stderr) => {
  console.log(stdout);
});
```

doesn't work.

the problem comes from the output

**Comment by titouancreach** [comment]
closed with commit a6302f3b722688df0620d565f1d94faecd35fad7
